src/Reflection/ReflectionClassConstant.php

```diff
@@ -219,7 +219,7 @@ public function getEndLine()
      */
     public static function create($class, $name)
     {
-        if (\class_exists('\\ReflectionClassConstant')) {
+        if (\class_exists(\ReflectionClassConstant::class)) {
```

Comment:
`::class` doesn’t work on 5.4.x which is still our minimum version :)

Also it’s super weird to me that PHP lets you do this. If the class doesn’t exist, which is what you’re checking for, _where did that class constant come from_.

Reply:
Oh, I thought the min version was 5.5 for some reason. Sorry!

Reply:
Maybe it's time to drop PHP 5.4 and 5.5 on `develop` now, anyway?